xml/System.Runtime.CompilerServices/MethodImplOptions.xml

```diff
@@ -120,7 +120,7 @@
       </ReturnValue>
       <MemberValue>512</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that the method contains hot path and should be optimised.</summary>
```

Comment:
I think it should be `a hot path`.
Even though `optimised` is correct, I think we need to stick with American english for the default documentation:

```suggestion
        <summary>Indicates that the method contains a hot path and should be optimized.</summary>
```